lib/importers/user_importer.rb

```diff
@@ -86,6 +76,27 @@ def self.update_user_from_auth(user, auth)
                 wiki_secret: auth.credentials.secret)
   end
 
+  def self.username_sanitization(username)
```

Comment:
Methods are usually named for what they do or what they return. In this case, I think `sanitize_username` is a good name.